markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -63,6 +63,26 @@ void MarkhorHWInterface::setupCTREDrive()
   current_limit_config.enable = true;
   current_limit_config.currentLimit = 33;
 
+  if(!nh.getParam("/markhor/tracks/markhor_tracks_node/kp", tracks_kp)){
+    ROS_WARN("TRACKS : Missing argument kp, assuming 0");
```

Comment:
Tell me what you think, but I feel like if we are missing PID value that is a critical error. I think we could do maybe a  `ROS_ERROR` or maybe even `ROS_FATAL` if one parameter of the PID is missing.

Reply:
Absolutely, does throwing a ros-fatal shuts down the node?

Reply:
Yup, I think so. Humm I was thinking about the flippers, maybe the tracks should just have an error since it's not robot-breaking-critical.

Reply:
I checked and it doesn't I implemented it with ros-shutdown, I think it should all be critical errors cause if we do not exit at that point, rhe can socket errors will flood and prevent the user from seeing what went wrong

Reply:
Alright I think that's fair. Ping @saxtot so he's in the loop on that. If you have something to add don't hesitate.

Reply:
Makes sense to me!